assets/js/modules/analytics/components/settings/SettingsView.js

```diff
@@ -41,6 +56,10 @@ export default function SettingsView() {
 			select( MODULES_TAGMANAGER ).getSingleAnalyticsPropertyID()
 	);
 
+	const dashboardView = useSelect( ( select ) =>
```

Comment:
If we use `isGA4DashboardView`, we don't need this.

Reply:
As mentioned below I think we should stick with `getDashboardView()` here for now.